### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by ArthDh**
### Proposed changes

Liz Rice Iframe was being rendered as code. Updated it to iframe.

### CODE REVIEW

**Comment by sudo-bmitch** [comment]
LGTM

**Comment by ArthDh** [comment]
Fixes #11986 #11978

File: get-started/index.md [hunk 1]
```diff
@@ -115,7 +115,7 @@ in Linux for a long time. Docker has worked to make these capabilities approacha
     a simple container, this talk doesn't go into networking, using images for the filesystem, 
     and more. But, it gives a _fantastic_ deep dive into how things are working.
 
-    <iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/8fi7uSYlOdc" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```
Comment (shaderecker) [inline get-started/index.md:118]: Any reason this white space before gyroscope was removed?

File: get-started/index.md [hunk 2]
```diff
@@ -115,7 +115,7 @@ in Linux for a long time. Docker has worked to make these capabilities approacha
...
+<iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/8fi7uSYlOdc" frameborder="0" allow="accelerometer; autoplay; encrypted-media;gyroscope; picture-in-picture" allowfullscreen></iframe>
```
Comment (shaderecker) [inline get-started/index.md:118]: Any reason this white space before gyroscope was removed?

Comment (ArthDh) [reply get-started/index.md]: It was unintentional, I have updated it in the latest commit.

**Comment by shaderecker** [comment]
Thanks for updating. LGTM.
I just cloned and locally built your patch. This does not fix #11978. so it must be a separate issue.

**Review by thaJeztah — APPROVED** [review]
LGTM, thanks!

**Pull request merged**